a3_finance/a3_finance/doctype/employee_time_loss/employee_time_loss.py

```
@@ -131,58 +131,65 @@ def validate(self):
+			def get_da_reference_month(month_num):
+				if month_num in [4, 5, 6]:
+					return 4  # April → Q1
+				elif month_num in [7, 8, 9]:
+					return 7  # July → Q2
+				elif month_num in [10, 11, 12]:
+					return 10  # October → Q3
+				else:
+					return 1  # Jan, Feb, Mar → Q4
+
+			da_month_number = get_da_reference_month(tl_month_number)
+			da_month = calendar.month_name[da_month_number]
```

Ruff (0.12.2)

Local variable `da_month` is assigned to but never used

Remove assignment to unused variable `da_month`

(F841)